dotcom-rendering/src/web/components/TextBlockComponent.tsx

```diff
@@ -35,33 +35,39 @@ const isOpenQuote = (t: string): boolean => {
 	].includes(t);
 };
 
-const isLongEnough = (html: string): boolean => {
-	// Only show a dropcap if the block of text is 200 characters or
-	// longer. But we first need to strip any markup from our html string so
-	// that we accurately measure the length that the reader will see. Eg. remove
-	// link tag html.
+const stripHtmlFromString = (html: string): string => {
 	// https://stackoverflow.com/questions/822452/strip-html-from-text-javascript is
 	// a good discussion on how this can be done. Of the two approaches, regex and
 	// DOM, both have unikely failure scenarios but the impact for failure with DOM
 	// manipulation carries a potential security risk so we're using a regex.
-	return html.replace(/(<([^>]+)>)/gi, '').length >= 199;
+	return html.replace(/(<([^>]+)>)/gi, '');
 };
 
-const decideDropCapLetter = (html: string): string => {
-	const first = html.substr(0, 1);
+const getDropCappedSentence = (html: string) => {
+	const first = html.substring(0, 1);
+
+	// If it starts with a quote and a letter, drop "“A"
 	if (isOpenQuote(first)) {
-		const second = html.substr(1, 1);
+		const second = html.substring(1, 1);
 
-		if (!isLetter(second)) {
-			return '';
+		if (isLetter(second)) {
+			return {
+				dropCap: `${first}${second}`,
+				restOfSentence: html.substring(2),
+			};
 		}
-		return `${first}${second}`;
 	}
 
-	return isLetter(first) ? first : '';
+	// Or just drop if the first is a letter
+	if (isLetter(first)) {
+		return {
+			dropCap: first,
+			restOfSentence: html.substring(1),
+		};
+	}
 };
 
-const allowsDropCaps = (format: ArticleFormat) => {
+const isValidDropCapFormat = (format: ArticleFormat) => {
```

Comment:
I've tried to make these methods more descriptive of what they're doing for example, there are other rules that `allowDropCaps`, so I refactored this to be the specific reason this allows drop caps.